src/components/header/Navigation.css

```diff
@@ -12,7 +12,7 @@
   z-index: 200;
```

Comment:
well I think z-index value can be lower down well if you need it anyways you can keep it as it is